aspnetcore/signalr/javascript-client.md

```diff
@@ -60,14 +58,14 @@ The client library is available on the following CDNs:
 
 The following code creates and starts a connection. The hub's name is case insensitive:
 
-[!code-javascript[](javascript-client/samples/3.x/SignalRChat/wwwroot/chat.js?range=3-6,29-45)]
+[!code-javascript[](javascript-client/samples/6.x/SignalRChat/wwwroot/js/chat.js)]
 
 ### Cross-origin connections
 
 Typically, browsers load connections from the same domain as the requested page. However, there are occasions when a connection to another domain is required.
 
 > [!IMPORTANT]
-> The client code must use an absolute URL instead of a relative URL. Change `.withUrl("/chathub")` to `.withUrl("https://myappurl/chathub")`.
+> The client code ***must*** use an absolute URL instead of a relative URL. Change `.withUrl("/chathub")` to `.withUrl("https://myappurl/chathub")`.
```

Comment:
If you must change to absolute, why do all the tutorials show relative and work? I deployed to Azure and it works for everyone. Or is the context for CORS?

Reply:
This section is talking about making a cross domain request, i.e. a request from bing.com to google.com. It might make sense to clarify that though.